golem/verificator/imgverifier.py

```diff
@@ -1,17 +1,19 @@
-from __future__ import division
 import logging
 import math
+from random import uniform
+from typing import Optional
 
-from .imgrepr import (ImgRepr, PILImgRepr)
+import numpy
+from ssim import compute_ssim
 
-from .verifier import SubtaskVerificationState
+from golem.verificator.constants import SubtaskVerificationState
+from golem.verificator.imgrepr import (ImgRepr, PILImgRepr)
```

Comment:
please use this file from different location and remove `golem.verificator.imgrepr`
```suggestion
from apps.rendering.resources import (ImgRepr, PILImgRepr)
```

Reply:
:+1: done